xcodeproj/internal/xcode_targets.bzl

```diff
@@ -327,11 +331,22 @@ def _set_bazel_outputs_product(
     if build_mode != "bazel":
         return
 
-    path = xcode_target.outputs.product_path
-    if not path:
-        return
-
-    build_settings["BAZEL_OUTPUTS_PRODUCT"] = path
+    product_file = xcode_target.outputs.product_file
+    if product_file:
+        build_settings["BAZEL_OUTPUTS_PRODUCT"] = product_file.path
+    
```

Comment:
This regressed. I changed the logic to use `xcode_target.outputs.product_path`.

Reply:
Fixed